spec/functions/ip_to_cron_spec.rb

```diff
@@ -1,24 +1,23 @@
 require 'spec_helper'
 
 # rubocop:disable RSpec/MultipleExpectations
-describe 'ip_to_cron', type: :puppet_function do
+describe 'ip_to_cron' do
   it 'exists' do
-    expect(Puppet::Parser::Functions.function('ip_to_cron')).to eq('function_ip_to_cron')
+    is_expected.not_to be_nil
   end
 
   describe 'when runinterval is 1 hour' do
     let(:runinterval) { 3600 }
 
     result = nil
     context 'and IP address is 10.0.0.150' do
-      before do
-        allow(scope).to receive(:lookupvar).with('ipaddress').and_return('10.0.0.150')
-      end
+      let(:facts) { { networking: { ip: '10.0.0.150' } } }
+
       it 'returns \'*\' for the hour' do
-        expect(subject.call([runinterval])[0]).to eq('*')
+        expect(subject.execute(runinterval)[0]).to eq('*')
       end
       it 'and one value between 0 and 59 for the minute' do
-        result = subject.call([runinterval])
+        result = subject.execute(runinterval)
```

Comment:
`call` does not behave as [documented](https://github.com/rodjek/rspec-puppet/issues/719).  The function was getting called, but with *no* arguments.  This meant it used the default `1800` seconds, not the `3600` intended.